code/datums/mind.dm

```diff
@@ -72,17 +73,29 @@
 	var/brigged_since = -1
 	var/suicided = FALSE
 
-	New(var/key)
-		src.key = key
-
 	//put this here for easier tracking ingame
 	var/datum/money_account/initial_account
 
 	//zealot_master is a reference to the mob that converted them into a zealot (for ease of investigation and such)
 	var/mob/living/carbon/human/zealot_master = null
 
+/datum/mind/New(var/key)
+	src.key = key
+
+
+/datum/mind/Destroy()
+	ticker.minds -= src
+	if(islist(antag_datums))

```

Comment:
According to [this thread](http://www.byond.com/forum/?post=273693) `islist` will always return 1 on this cause you have `var/list/antag_datums = list()` earlier up in the initialization stuff.

Reply:
@tigercat2000 thoughts.

Reply:
I think that's fine; I'm pretty sure it only cares about this if you explicitly set `antag_datums` to something like `null` or equal to a number or another datum; then it would fail.